From a6a26139e4bd07cb780fe1ab2bcba3f57b13726a Mon Sep 17 00:00:00 2001
From: Kathiravan T <quic_kathirav@quicinc.com>
Date: Tue, 18 Jul 2023 14:49:53 +0530
Subject: [PATCH 385/500] arm: qcom: enable ARM_GIC_V2M if PCI is defined

Enable ARM_GIC_V2M if PCI is enabled in ARCH_QCOM.

Change-Id: I8bd4050d026c5a74d4ad31a409d6e326294aab7a
Signed-off-by: Kathiravan T <quic_kathirav@quicinc.com>
---
 arch/arm/mach-qcom/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-qcom/Kconfig b/arch/arm/mach-qcom/Kconfig
index 06b7b9c1a8f1..6a8be97e905a 100644
--- a/arch/arm/mach-qcom/Kconfig
+++ b/arch/arm/mach-qcom/Kconfig
@@ -6,6 +6,7 @@ menuconfig ARCH_QCOM
 	select ARM_AMBA
 	select PINCTRL
 	select QCOM_SCM if SMP
+	select ARM_GIC_V2M if PCI
 	help
 	  Support for Qualcomm's devicetree based systems.
 
-- 
2.34.1

